### ISSUE TITLE
Can't load GLTF with shape-keys animations

### ISSUE THREAD

**Issue by strangetype** [jnsmalm/pixi3d#176]
PIXI3D can't load GLTF file with shape keys animation:
![image](https://user-images.githubusercontent.com/4980343/231093691-dbc49a71-00db-4e9f-b8c8-4f2ce3c4ca1d.png)
errors:
![image](https://user-images.githubusercontent.com/4980343/231093786-e0eb2554-c167-4b99-8362-c3e7c107e49d.png)
shader error:
![image](https://user-images.githubusercontent.com/4980343/231093950-914a2a94-0e0d-44db-9773-f42b9a60faf5.png)

**Comment by jnsmalm** [comment]
Do you have an example glTF file so I can debug?

**Comment by strangetype** [comment]
> 

yes, [this file](https://drive.google.com/file/d/1il2DOgSc7bE3kXJUJgatIGotRohefegY/view?usp=sharing)

**Comment by jnsmalm** [comment]
Sorry for taking so long, I forgot about this one. I have been looking into this issue and there is no quick fix for this. The implementation has to be changed to handle these many morph targets (150 of them).

A question, are you sure you need to use morph targets to do this kind of animation or was this file just to demo the issue in Pixi3D? If not, you can create this animation just using transformation animation instead.

**Comment by strangetype** [comment]
I need morph targets because this kind of animation creatied by physics emulation. there is no any 3d editors which is able to create physics animation using transform animation. this file is not for demo.

**Comment by jnsmalm** [comment]
Would it be possible to instead use joints/bones for the physics simulation and export those instead? Thing is I'm pretty occupied with other stuff right now and don't have time to implement support for more morph targets.

**Comment by strangetype** [comment]
thx for advice.
